integration-tests/utils/integration-helpers.js

```diff
@@ -37,11 +37,15 @@ const makeWorkspace = withSignedInPage(async ({ page, billingProject }) => {
 
 
 const deleteWorkspace = withSignedInPage(async ({ page, billingProject, workspaceName }) => {
-  await page.evaluate((name, billingProject) => {
-    return window.Ajax().Workspaces.workspace(billingProject, name).delete()
-  }, workspaceName, billingProject)
+  try {
+    await page.evaluate((name, billingProject) => {
+      return window.Ajax().Workspaces.workspace(billingProject, name).delete()
+    }, workspaceName, billingProject)
 
-  rawConsole.info(`deleted workspace: ${workspaceName}`)
+    rawConsole.info(`deleted workspace: ${workspaceName}`)
+  } catch (e) {
+    console.error(`Failed to delete workspace: ${workspaceName}. Error: ${e.message}`)
```

Comment:
How would someone know to look for this? Should we fail the test?

Note also we typically use rawConsole, but perhaps the usage of console here was intentional to make this stand out more (you will get a bit of a stack trace)?

Reply:
I added this in for debugging since I thought the delete was failing and obscuring an actual test error. It turns out the `Evaluation failed` error was in creating a workspace (seen 1/100 times) so I updated that to throw a more descriptive error message and changed the delete workspace function to do the same thing.